runtime/rococo/src/lib.rs

```diff
@@ -103,7 +103,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("rococo"),
 	impl_name: create_runtime_str!("parity-rococo-v1-1"),
 	authoring_version: 0,
-	spec_version: 210,
+	spec_version: 215,
```

Comment:
do we have to look at the rococo branch https://github.com/paritytech/polkadot/blob/575a89e1ca6703a0b12e30c01ba8a1e4a5cfed84/runtime/rococo/src/lib.rs#L104
to verify it's correct? :P